From 9d96c47f6b922ba02dee62337236f1b87074cfe0 Mon Sep 17 00:00:00 2001
From: Romaric Jodin <rjodin@google.com>
Date: Thu, 28 Aug 2025 18:56:33 +0200
Subject: [PATCH 1/3] libclc: CMake: include GetClangResourceDir (#155836)

`get_clang_resource_dir` is not guarantee to be there. Make sure of it
by including `GetClangResourceDir`.
---
 libclc/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libclc/CMakeLists.txt b/libclc/CMakeLists.txt
index 756e097dcf12..5c4e12793329 100644
--- a/libclc/CMakeLists.txt
+++ b/libclc/CMakeLists.txt
@@ -99,6 +99,7 @@ else()
 
   # Setup the paths where libclc runtimes should be stored. By default, in an
   # in-tree build we place the libraries in clang's resource driectory.
+  include(GetClangResourceDir)
   get_clang_resource_dir( LIBCLC_OUTPUT_DIR PREFIX ${LLVM_LIBRARY_OUTPUT_INTDIR}/.. )
 
   # Note we do not adhere to LLVM_ENABLE_PER_TARGET_RUNTIME_DIR.
-- 
2.51.0.384.g4c02a37b29-goog

